api/pom.xml

```diff
@@ -365,6 +475,7 @@ Use is subject to <a href="{@docRoot}/doc-files/speclicense.html" target="_top">
                     <docfilessubdirs>true</docfilessubdirs>
                     <javadocDirectory>${main.basedir}/activation/src/main/javadoc</javadocDirectory>
                     <detectJavaApiLink>false</detectJavaApiLink>
+                    <release>11</release>
```

Comment:
double check the `${current.year}` prop is resolved properly on generated javadoc as well as pointer to speclicense file works

Reply:
This looks ok:
Comments to : jaf-dev@eclipse.org.
Copyright © 2019, 2021 Eclipse Foundation. All rights reserved.
Use is subject to license terms.
Locally I got file:///data/metro/jaf/api/target/apidocs/doc-files/speclicense.html URL for speclicense. The same pattern is being used in JAX-WS API so hope it's OK.